synthesizer/src/process/stack/finalize_registers/load.rs

```diff
@@ -76,28 +47,17 @@ impl<N: Network> FinalizeRegisters<N> {
             // If the register is a locator, then return the stack value.
             Register::Locator(..) => stack_value.clone(),
             // If the register is a register member, then load the specific stack value.
-            Register::Member(_, ref path) => {
-                match stack_value {
-                    // Retrieve the plaintext member from the path.
-                    Value::Plaintext(plaintext) => Value::Plaintext(plaintext.find(path)?),
-                    // Retrieve the record entry from the path.
-                    Value::Record(record) => match record.find(path)? {
-                        Entry::Constant(plaintext) | Entry::Public(plaintext) | Entry::Private(plaintext) => {
-                            Value::Plaintext(plaintext)
-                        }
-                    },
-                }
-            }
+            Register::Member(_, ref path) => stack_value.find(path)?,
         };
 
         // Retrieve the register type.
         match self.finalize_types.get_type(stack, register) {
             // Ensure the stack value matches the register type.
-            Ok(register_type) => stack.matches_register_type(&stack_value, &register_type)?,
```

Comment:
For safety, does it make sense to enforce the record case here?

Reply:
```rs
        // Ensure that the stack value is a plaintext value.
        let stack_value = match stack_value {
            Value::Plaintext(plaintext) => plaintext,
            Value::Record(_) => bail!("Cannot load a record to a finalize register"),
        };
```

Reply:
I don't think we need to since:
- L54 always produces a plaintext type.
- `stack_value` is a plaintext value.
Will rename `stack_value` to `plaintext_value` and `register_type` to `plaintext_type` for clarity.